### PR TITLE
Use support.md instead of direct links to GitHub

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This is a response to https://github.com/pingcap/docs/pull/13375/files#r1308409173

We do still want users to report issues on GitHub, but for questions we have Slack/Discourse/etc and for bugs we want TiDB Enterprise and TiDB Cloud users to create a support ticket as well.

And for some services we have moved from Zendesk to Jira ServiceDesk.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14648
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ summary: 在使用 TiDB 时遇到问题，如何获取支持。
     - [AskTUG 论坛](https://asktug.com/)
     - [Stack Overflow](https://stackoverflow.com/questions/tagged/tidb)（在 #tidb 标签下提问）
 
++ 报告 Bug：
```
Comment (Oreoxmt) [inline support.md:19]:
```suggestion
+ 报告 bug：
```

File: tidb-resource-control.md [hunk 2]
```diff
@@ -344,7 +344,7 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
 
 > **警告：**
 >
-> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues/new/choose) 反馈。
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请[提交 issue](/support.md) 反馈。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:347]: - 这里英文文档还是使用 GitHub issue 链接 https://github.com/pingcap/docs/blob/c1e08010a4a1439c93417ece933ac177dc881ea2/tidb-resource-control.md?plain=1#L395-L399
- troubleshoot-tidb-cluster.md 未更新：https://github.com/pingcap/docs/blob/c1e08010a4a1439c93417ece933ac177dc881ea2/troubleshoot-tidb-cluster.md?plain=1#L9

Comment (ran-huang) [reply tidb-resource-control.md:347]: troubleshoot-tidb-cluster.md 加上了，英文：https://github.com/pingcap/docs/pull/15057

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**